### PR TITLE
fix: change references from master to main

### PR DESCRIPTION
**Pull request by efraa**

### CODE REVIEW

File: mods/ctl/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Command-Line for for Fonoster
 [![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
 [![Version](https://img.shields.io/npm/v/ctl.svg)](https://npmjs.org/package/ctl)
 [![Downloads/week](https://img.shields.io/npm/dw/ctl.svg)](https://npmjs.org/package/ctl)
-[![License](https://img.shields.io/npm/l/ctl.svg)](https://github.com/fonoster/fonoster/blob/master/package.json)
```
Comment (psanders) [inline mods/ctl/README.md:8]: Oclif automatically generates this file. This change will get lost unless we fix the script that generates the README.

Comment (efraa) [reply mods/ctl/README.md:8]: 😳

**Review by psanders — APPROVED** [review]
Awesome. Thanks!

**Review by psanders — COMMENTED** [review]
LGTM.

**Pull request merged**